### PR TITLE
Added domain field to user

### PR DESCRIPTION
**Pull request by thomaspatzke**

### CODE REVIEW

File: schemas/user.yml [hunk 1]
```diff
@@ -56,3 +56,10 @@
 
         Useful if `user.id` or `user.name` contain confidential information and
         cannot be used.
+
+    - name: domain
+      level: extended
+      type: keyword
+      example: domainname
```
Comment (webmat) [inline schemas/user.yml]: I'm not a fan of this example, as it doesn't add any information, or doesn't jog the user's memory in giving a domain name that people may recognize.

I would either remove the example, or perhaps replace with something else that's commonly seen as an LDAP or AD domain name. Not sure what that would be, though. "corp", "intranet", "acme"?

File: schemas/user.yml [hunk 2]
```diff
@@ -56,3 +56,10 @@
...
+      description: >
+        Name of user domain, e.g. Active Directory or LDAP domain name.
```
Comment (webmat) [inline schemas/user.yml]: Can you adjust the description to the following, please? Note that the two paragraphs will give us a "short" description automatically, here.

"Name of the directory the user is a member of.

For example, an LDAP or Active Directory domain name."

File: schemas/user.yml (hunk 1 above)
Comment (thomaspatzke) [reply schemas/user.yml]: Removed this, I think it should be obvious for users of this field.

File: schemas/user.yml (hunk 2 above)
Comment (thomaspatzke) [reply schemas/user.yml]: Changed it. Update PR in few seconds.

**Comment by thomaspatzke** [comment]
Implemented changes as proposed.

**Comment by webmat** [comment]
The field description in all the files is based on what's in `schemas/*.yml`. Please run `make generate` to re-generate all other files. That's why the build is breaking :-)

**Comment by thomaspatzke** [comment]
Sorry, forgot this! Just pushed the commited build.

**Comment by webmat** [comment]
@thomaspatzke Thanks for your contribution :-)

**Comment by webmat** [comment]
And of course I thought "oh wait, what about the Changelog" one millisecond after merging 😂

**Pull request merged**